daemon/metrics.go

```diff
@@ -26,12 +26,13 @@ func init() {
 		containerActions.WithValues(a).Update(0)
 	}
 	networkActions = ns.NewLabeledTimer("network_actions", "The number of seconds it takes to process each network action", "action")
-	engineVersion = ns.NewLabeledGauge("engine", "The version and commit information for the engine process", metrics.Unit("info"),
+	engineInfo = ns.NewLabeledGauge("engine", "The information related to the engine and the OS it is running on", metrics.Unit("info"),
 		"version",
 		"commit",
 		"architecture",
-		"graph_driver", "kernel",
-		"os",
+		"graphdriver",
+		"kernel", "os",
+		"os_type", "id", // ID is a randomly generated unique identifier (e.g. UUID4)
```

Comment:
@rogaha Yes:

```console
$ docker info | grep ID
 NodeID: t2oskt6v6jb74t4a6av8f505s
 ClusterID: k1q17rn7zptkc95s6cxyj34ug
ID: 5GPU:Q5YR:5RQP:46KW:BLDN:4MSK:4K4R:LDKX:4LGJ:XBHB:AMOT:FBKK
```

Right now, it is a unique, cryptographic identity set at startup, disconnected with the swarm node id. At some point, we should join these concepts, but, for now, this probably the best bet.

We can make this smaller by just generating with `swarmkit/identity` and storing it in a file somewhere, but that is for another PR. It will be easiest if this matches with the `docker info` ID.